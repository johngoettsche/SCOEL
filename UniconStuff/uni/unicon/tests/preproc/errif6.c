/* test error message: #elif with no #if */

#if 1
#endif
#elif 1
