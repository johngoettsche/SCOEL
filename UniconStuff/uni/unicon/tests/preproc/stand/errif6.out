pp: File errif6.c; Line 5: invalid context for #elif
